lms/djangoapps/verify_student/views.py

```diff
@@ -1213,3 +1214,20 @@ def get(self, request):
         """
         IDV_workflow = IDVerificationService.get_verify_location()
         return redirect(IDV_workflow)
+
+
+class PhotoUrlsView(APIView):
```

Comment:
This doesn't have a serializer because I was trying to follow the pattern of the other views in this file, but I can update this if it is the better option